AppleStore/AppleStore/SearchViewController.swift

```diff
@@ -22,6 +22,7 @@ final class SearchViewController: UIViewController {
     private let secondImageSearchImageView = UIImageView()
     private let thirdImageSearchImageView = UIImageView()
     private let fourthImageSearchImageView = UIImageView()
+    private let productsScrollView = UIScrollView()
 
     // MARK: - Public Properties
     let firstView = UIView()
```

Comment:
```suggestion
  private  let firstView = UIView()
```
все ui приватными делаем в них из вне не передаем